acpt.php

```diff
@@ -7,15 +7,14 @@
 Author URI: http://iambrian.com/
 */
 
-if ( ! defined( 'ABSPATH' ) )
```

Comment:
Bracket standards = always use brackets, or alternate. Where does it say not newline?